components/blog/BlogPost.vue

```diff
@@ -1,88 +1,78 @@
 <template>
-  <b-row
-    class="flex-md-row"
-    data-qa="blog post"
-  >
-    <b-col
-      cols="12"
-    >
-      <article>
-        <div class="title">
-          <h1 data-qa="blog post title">
-            {{ title }}
-          </h1>
-          <p
-            v-if="description"
-            class="lead"
-          >
-            {{ description }}
-          </p>
-        </div>
-      </article>
-    </b-col>
-    <b-col cols="12">
-      <ImageWithAttribution
-        v-if="heroImage"
-        :src="heroImage.url"
-        :image-content-type="heroImage.contentType"
-        :rights-statement="hero.license"
-        :attribution="hero"
-        hero
-      />
-    </b-col>
-    <b-col
-      cols="12"
-    >
-      <article>
-        <div class="font-small font-weight-bold d-block">
-          <time
-            v-if="datePublished"
-            class="d-inline-block"
-            data-qa="date"
-            :datetime="datePublished"
-          >
-            {{ $t('blog.published', { date: $d(new Date(datePublished), 'short') }) }}
-          </time>
-          <span
-            v-if="authors"
-          >
-            {{ $t('blog.by') }}
-          </span>
-          <BlogAuthor
-            v-for="(author, index) in authors"
-            :key="index"
-            class="d-inline-block"
-            :name="author.name"
-            :organisation="author.affiliation"
-            :url="author.url"
+  <b-row class="flex-md-row">
```

Comment:
Makes this a `<div>` without any class or add no-gutters to the row, like `<b-row class="flex-md-row" no-gutters>` to avoid adding an overflow-x. I prefer the `<div>`.

Reply:
yes good suggestion, this solves the overflow problem and is much cleaner